h/templates/includes/annotation_card.html.jinja2

```diff
@@ -62,7 +62,14 @@
   <footer class="annotation-card__footer">
     {% if presented_annotation.incontext_link %}
       <a href="{{ presented_annotation.incontext_link }}" target="_blank" title="{% trans %}Visit annotation in context{% endtrans %}">
-        {{ svg_icon('up-right-arrow', 'annotation-card__incontext-link') }}
+        {{ svg_icon('up-right-arrow', 'annotation-card__footer-link annotation-card__incontext-link') }}
+      </a>
+      <a href="#" share-widget-config='{
```

Comment:
I wasn't sure if new lines were legal in HTML attribute values but apparently [they are](http://stackoverflow.com/questions/8047042/)